docs/querying/basics.md

```diff
@@ -81,6 +81,16 @@ Examples:
     0x8f
     -Inf
     NaN
+	
+
+As of version 2.52, float values can also be represented using the syntax of durations, where the duration literal is converted into a float value corresponding to the number of seconds the duration literal represents.
+
+Examples:
+
+    1s // === 1.0
+    2m // == 120.0
+    1ms // == 0.01
```

Comment:
Comments in PromQL use a `>`. And maybe use English rather than an operator from another programming language. E.g. `1s # Equivalent to 1.0.`